windows全屏下拖动标题栏偏下部能拖动窗口并且出现窗口抖动

### 问题描述

如题，全屏模式能拖动窗口

### 复现步骤

全屏下拖动标题栏下部分区域可拖动窗口且出现卡顿抖动

https://github.com/user-attachments/assets/4512311e-dd1c-4f1e-9da9-b8d37e4c27f6



### Ani 版本号

3.11.0-b4

### 操作系统

Windows

### 应用日志

```text
2024-10-03 19:53:13,604 [INFO ] Ani: Ani platform: Windows, version: 3.11.0-beta04
2024-10-03 19:53:13,942 [INFO ] app.desktop.WindowsSingleInstanceChecker: Application is running with mutex to prevent double instances: AniAppSingleInstanceMutex
2024-10-03 19:53:13,943 [INFO ] Ani: Single instance check took 176.380900ms
2024-10-03 19:53:14,105 [INFO ] app.platform.WindowsSystemProxyDetector: Detected system proxy: http://127.0.0.1:7890
2024-10-03 19:53:14,243 [INFO ] app.data.source.media.source.subscription.MediaSourceSubscriptionUpdater: MediaSourceSubscriptionUpdater.updateAllOutdated
2024-10-03 19:53:14,306 [INFO ] Ani: TorrentManager saveDir: C:\Users\MINEc\AppData\Local\Him188\Ani\cache\torrent-data
2024-10-03 19:53:14,333 [INFO ] app.torrent.anitorrent.AnitorrentLibraryLoader: Loading library vcruntime140 from: vcruntime140.dll
2024-10-03 19:53:14,334 [INFO ] app.torrent.anitorrent.AnitorrentLibraryLoader: Loading library vcruntime140_1 from: vcruntime140_1.dll
2024-10-03 19:53:14,336 [INFO ] app.torrent.anitorrent.AnitorrentLibraryLoader: Loading library MSVCP140 from: msvcp140.dll
2024-10-03 19:53:14,337 [INFO ] app.torrent.anitorrent.AnitorrentLibraryLoader: Loading library LIB_EAY_RELEASE_0 from: libcrypto-3-x64.dll
2024-10-03 19:53:14,337 [ERROR] app.data.source.media.fetch.MediaSourceManager: MediaSourceFactory 'local-file-system' not found for local-file-system
2024-10-03 19:53:14,342 [INFO ] app.torrent.anitorrent.AnitorrentLibraryLoader: Loading library SSL_EAY_RELEASE_0 from: libssl-3-x64.dll
2024-10-03 19:53:14,344 [INFO ] app.torrent.anitorrent.AnitorrentLibraryLoader: Loading library LIBTORRENT_RASTERBAR from: torrent-rasterbar.dll
2024-10-03 19:53:14,347 [INFO ] app.torrent.anitorrent.AnitorrentLibraryLoader: Loading library ANITORRENT from: anitorrent.dll
2024-10-03 19:53:14,349 [INFO ] app.tools.torrent.engines.AnitorrentEngine: Loaded libraries for AnitorrentEngine
2024-10-03 19:53:14,350 [INFO ] app.tools.torrent.engines.AnitorrentEngine: Loaded libraries for AnitorrentEngine
2024-10-03 19:53:14,352 [ERROR] app.data.source.media.fetch.MediaSourceManager: MediaSourceFactory 'nyafun' not found for nyafun
2024-10-03 19:53:14,354 [INFO ] app.torrent.anitorrent.AnitorrentTorrentDownloader: Creating a new AnitorrentTorrentDownloader
2024-10-03 19:53:14,356 [ERROR] app.data.source.media.fetch.MediaSourceManager: MediaSourceFactory 'mxdongman' not found for mxdongman
2024-10-03 19:53:14,368 [INFO ] app.torrent.anitorrent.AnitorrentTorrentDownloader: AnitorrentTorrentDownloader created
2024-10-03 19:53:15,032 [INFO ] Ani: renderApi: DIRECT3D
2024-10-03 19:53:15,195 [TRACE] app.data.source.session.SessionManager: requireOnline
2024-10-03 19:53:15,523 [INFO ] app.ui.update.AutoUpdateViewModel: Checking latest version, updateSettings=UpdateSettings(autoCheckUpdate=true, releaseClass=alpha, autoDownloadUpdate=true, inAppDownload=true, _placeholder=0)
2024-10-03 19:53:15,967 [INFO ] app.tools.caching.LazyDataCacheImpl: Initialize LazyDataCache(collectionsByType-DOING) with save LazyDataCacheSave(page=1, totalSize=22, time=1727955905478, list=[SubjectCollectionItem(DDDD   ħ   ƻ ), SubjectCollectionItem(ƾ    ), SubjectCollectionItem(       -  ֿ    ʱ  -), SubjectCollectionItem(2.5  Ԫ   ջ ), SubjectCollectionItem(ҹ     ), SubjectCollectionItem(     Ϊħ    Ů), SubjectCollectionItem(    ħ      Լ  ), SubjectCollectionItem(    ϵ    ⴫  &   ＾), SubjectCollectionItem(                       ), SubjectCollectionItem(Ե      ), SubjectCollectionItem(ҹӣ ҵĴ   ս), SubjectCollectionItem(ɱ  Ԣ  ), SubjectCollectionItem(  ѻ      ), SubjectCollectionItem(       Զ ĵط ), SubjectCollectionItem(δ   ռ ), SubjectCollectionItem(    ʦ   ֮  ), SubjectCollectionItem( Թ   ), SubjectCollectionItem(С ƹ Basue), SubjectCollectionItem( ж   ҲҪ̸      ), SubjectCollectionItem(    '), SubjectCollectionItem(    ʯ֮  ), SubjectCollectionItem( Ǽ ţ  )])
2024-10-03 19:53:16,128 [INFO ] app.ui.update.AutoUpdateViewModel: Checking latest version, updateSettings=UpdateSettings(autoCheckUpdate=true, releaseClass=alpha, autoDownloadUpdate=true, inAppDownload=true, _placeholder=0)
2024-10-03 19:53:16,234 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/me [Authorized]: 200 OK in 923.747500ms
2024-10-03 19:53:17,578 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/jojoc2h4/collections?subject_type=2&type=3&limit=10&offset=0 [Authorized]: 200 OK in 623.843600ms
2024-10-03 19:53:18,293 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/subjects/431370 [Authorized]: 200 OK in 623.022300ms
2024-10-03 19:53:18,293 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/-/collections/431370/episodes?offset=0&limit=100&episode_type=0 [Authorized]: 200 OK in 616.547100ms
2024-10-03 19:53:18,928 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/subjects/115932 [Authorized]: 200 OK in 611.229300ms
2024-10-03 19:53:18,939 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/-/collections/115932/episodes?offset=0&limit=100&episode_type=0 [Authorized]: 200 OK in 622.457200ms
2024-10-03 19:53:19,549 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/subjects/495562 [Authorized]: 200 OK in 607.066400ms
2024-10-03 19:53:19,556 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/-/collections/495562/episodes?offset=0&limit=100&episode_type=0 [Authorized]: 200 OK in 613.626400ms
2024-10-03 19:53:20,207 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/-/collections/397604/episodes?offset=0&limit=100&episode_type=0 [Authorized]: 200 OK in 621.805300ms
2024-10-03 19:53:20,929 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/subjects/397604 [Authorized]: 200 OK in 1.344374700s
2024-10-03 19:53:21,534 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/subjects/410346 [Authorized]: 200 OK in 597.356600ms
2024-10-03 19:53:21,582 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/-/collections/410346/episodes?offset=0&limit=100&episode_type=0 [Authorized]: 200 OK in 646.257500ms
2024-10-03 19:53:22,218 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/-/collections/475555/episodes?offset=0&limit=100&episode_type=0 [Authorized]: 200 OK in 610.333200ms
2024-10-03 19:53:22,251 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/subjects/475555 [Authorized]: 200 OK in 643.502700ms
2024-10-03 19:53:22,405 [INFO ] app.ui.update.UpdateChecker: Got latest version from CN server: null
2024-10-03 19:53:22,878 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/subjects/424663 [Authorized]: 200 OK in 612.807200ms
2024-10-03 19:53:22,885 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/-/collections/424663/episodes?offset=0&limit=100&episode_type=0 [Authorized]: 200 OK in 619.754500ms
2024-10-03 19:53:23,504 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/subjects/100205 [Authorized]: 200 OK in 607.957300ms
2024-10-03 19:53:23,510 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/-/collections/100205/episodes?offset=0&limit=100&episode_type=0 [Authorized]: 200 OK in 613.946200ms
2024-10-03 19:53:24,155 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/subjects/475354 [Authorized]: 200 OK in 607.79ms
2024-10-03 19:53:24,155 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/-/collections/475354/episodes?offset=0&limit=100&episode_type=0 [Authorized]: 200 OK in 607.691700ms
2024-10-03 19:53:24,793 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/users/-/collections/372010/episodes?offset=0&limit=100&episode_type=0 [Authorized]: 200 OK in 619.627500ms
2024-10-03 19:53:25,268 [INFO ] datasources.bangumi.BangumiClientImpl:   GET https://api.bgm.tv/v0/subjects/372010 [Authorized]: 200 OK in 1.094756500s
```

--- a/app/shared/ui-foundation/src/desktopMain/kotlin/ui/foundation/interaction/WindowDragArea.desktop.kt
+++ b/app/shared/ui-foundation/src/desktopMain/kotlin/ui/foundation/interaction/WindowDragArea.desktop.kt
@@ -13,6 +13,8 @@ import androidx.compose.foundation.window.WindowDraggableArea
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import me.him188.ani.app.ui.foundation.layout.LocalPlatformWindow
+import me.him188.ani.utils.platform.Platform
+import me.him188.ani.utils.platform.currentPlatform
 
 @Composable
 actual inline fun WindowDragArea(
@@ -20,9 +22,13 @@ actual inline fun WindowDragArea(
     crossinline content: @Composable () -> Unit
 ) {
     val windowScope = LocalPlatformWindow.current.windowScope
-    windowScope?.run {
-        WindowDraggableArea(modifier) {
-            content()
-        }
-    } ?: content()
+    if (currentPlatform() is Platform.Windows) {
+        content()
+    } else {
+        windowScope?.run {
+            WindowDraggableArea(modifier) {
+                content()
+            }
+        } ?: content()
+    }
 }

--- a/app/shared/ui-foundation/src/desktopMain/kotlin/ui/foundation/interaction/WindowDragArea.desktop.kt
+++ b/app/shared/ui-foundation/src/desktopMain/kotlin/ui/foundation/interaction/WindowDragArea.desktop.kt
@@ -12,9 +12,9 @@ package me.him188.ani.app.ui.foundation.interaction
 import androidx.compose.foundation.window.WindowDraggableArea
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import me.him188.ani.app.ui.foundation.LocalPlatform
 import me.him188.ani.app.ui.foundation.layout.LocalPlatformWindow
 import me.him188.ani.utils.platform.Platform
-import me.him188.ani.utils.platform.currentPlatform
 
 @Composable
 actual inline fun WindowDragArea(
@@ -22,7 +22,7 @@ actual inline fun WindowDragArea(
     crossinline content: @Composable () -> Unit
 ) {
     val windowScope = LocalPlatformWindow.current.windowScope
-    if (currentPlatform() is Platform.Windows) {
+    if (LocalPlatform.current is Platform.Windows) {
         content()
     } else {
         windowScope?.run {

